web-client/src/presenter/computeds/completeDocumentTypeSectionHelper.ts

```diff
@@ -8,6 +8,24 @@ import { state } from '@web-client/presenter/app.cerebral';
 
 import { ClientApplicationContext } from '@web-client/applicationContext';
 import { Get } from 'cerebral';
+import { PublicCase } from '@shared/business/entities/cases/PublicCase';
+
+export const isFirstIrsFiling = (
```

Comment:
Should this be a static method off of an entity? It seems like it's enforcing some business rules.

 I love that it's extracted out, super easy to read and understand!

Reply:
Done 👍🏻 made it a static method on the case entity. Seemed natural given that everything in there was based on the case that was being passed in.